stylesheets/app.css

```diff
@@ -0,0 +1,640 @@
+/* stylelint-disable indentation */
+
+body {
+  background-color: #ececeb;
+  width: 100%;
+}
+
+.container {
+  background-color: white;
+  padding: 0;
+}
+
+.main-content {
+  padding: 20px;
+}
+
+.main-content h3:hover {
+  text-decoration: underline red;
+}
+
+.nav-left-cont {
+  width: 300px;
+  padding-left: 26px;
+  padding-top: 14px;
+}
+
+.cont-logo {
+  width: 500px;
+  overflow: hidden;
+}
+
+.news-logo {
+  width: 100%;
+  height: 100px;
+}
+
+.text-white {
+  word-spacing: 5px;
+  line-height: normal;
+}
+
+.text-1 {
+  font-weight: 900;
+}
+
+.btn-log {
+  font-size: 18px;
+  font-weight: 900;
+  color: white;
+}
+
+.btn-log:hover {
+  text-decoration: none;
+}
+
+.btn-warning {
+  width: 40%;
+  font-size: 16px;
+  font-weight: bolder;
+  border-radius: 28px;
+  height: 10px;
+  box-sizing: content-box;
+  text-align: center;
+  line-height: normal;
+}
+
+.fa-user {
+  display: none;
+}
+
+.hd-btns {
+  margin: inherit;
+  width: 300px;
+  height: auto;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.fa-bars {
+  display: none;
+}
```

Comment:
Same style definition of the selector on line 67, group selectors with the same style definition